docs/arch/002-store-responses-in-query-string.md

```diff
@@ -0,0 +1,61 @@
+# ADR 2: Store responses in query string
+
+## Background
+
+At the time of writing Smart Answer's has two methods of storing user responses: appending to the URL path or saving to the user's session cookie.
```

Comment:
Would using `answer` instead of `response` be better I wonder?

Reply:
Do you mean:
```suggestion
At the time of writing Smart Answer's has two methods of storing user answers: appending to the URL path or saving to the user's session cookie.
```

Is the use of "response" confusing? Happy to update if so, otherwise if it still makes sense I'll leave it.


```suggestion
At the time of writing Smart Answer's has two methods of storing user responses: appending to the URL path or saving to the user's session cookie.
```

Reply:
No, not confused - but there is a subtle semantic difference.  A response can be nothing - you can choose not to respond or do so in a non-written/verbal way (smile, laugh, frown or shrug for example).  This seems to make "response" feel somewhat optional in nature.  "Answer" on the other hand is demanding and feels more urgent and  mandatory in nature.  In smart-answers the user has no choice.  If they want the results - they must answer the questions.  I don't think this should block, so I'll approve and resolve this conversation.